tensorboard/components/tf_backend/test/requestManagerTests.ts

```diff
@@ -289,6 +289,81 @@ describe('backend', () => {
         // resolving the first request triggers finishTheTest
       });
     });
+
+    it('throws InvalidRequestOptionsError on bad RequestOptions', function() {
```

Comment:
I had to think a bit reading this. Can you please change this to say `throws an error when a GET request has a body` instead? (or something to that effect)